windows/win_file.py

```diff
@@ -39,27 +39,46 @@
     required: true
     default: []
     aliases: ['dest', 'name']
+  target:
+    description:
+      - 'path for a Windows shortcut (only applies to c(state=shortcut))'
+    required: false
+    default: []
+    version_added: 2.2
+  arguments:
+    description:
+      - 'arguments for a Windows shortcut (only applies to c(state=shortcut))'
```

Comment:
It would be good to elaborate what the possible arguments could actually be, so that ansible users don't immediately start having to dig up msdn documentation to set extra arguments.  If there's some common arguments that could be handled via other ansible parameters that would be nice too (I haven't looked).